src/Blockfrost.Api/Services/ICardanoService.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Blockfrost.Api
+{
+    public interface ICardanoService :
+        IBlockfrostService,
+        IAccountService,
+        IAddressService,
+        IAssetService,
+        IBlockService,
+        IEpochService,
+        IPoolService,
+        ILedgerService,
+        IMetadataService,
+        ITransactionService
+    {
+        IAccountService Accounts { get; }
+        IAddressService Addresses { get; }
+        IAssetService Assets { get; }
+        IBlockService Blocks { get; }
+        IEpochService Epochs { get; }
+        ILedgerService Ledger { get; }
+        IMetadataService Metadata { get; }
+        IPoolService Pools { get; }
+        ITransactionService Transactions { get; }
+    }
+}
```

Comment:
This explains this PR best... so I'll mark this for anyone that is especially interested